AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift

```diff
@@ -0,0 +1,81 @@
+// Created by Lunar on 10/11/2021.
+//
+import Foundation
+import SystemConfiguration.CaptiveNetwork
+
+class CreateSessionDetailsViewModel: ObservableObject {
+    
+    @Published var sessionName: String = ""
+    @Published var sessionTags: String = ""
+    @Published var isIndoor = true
+    @Published var isWiFi = true
+    @Published var wifiPassword: String = ""
+    @Published var wifiSSID: String = ""
+    @Published var isConfirmCreatingSessionActive: Bool = false
+    @Published var isLocationSessionDetailsActive: Bool = false
+    @Published var showingAlert = false
+    @Published var isSSIDTextfieldDisplayed: Bool = false
+    
+    let baseURL: BaseURLProvider
+    
+    init(baseURL: BaseURLProvider) {
+        self.baseURL = baseURL
+    }
+    
+    func onScreenEnter() {
+        if let ssid = getWiFiSsid() { wifiSSID = ssid }
+        isSSIDTextfieldDisplayed = wifiSSID.isEmpty
+    }
+    
+    func onContinueClick(sessionContext: CreateSessionContext) -> CreateSessionContext {
```

Comment:
this is a bit weird that this is modifying a context instead of handing it end-to-end. But I think this is how this flow works, so it's required. Maybe a comment about that would be nice so if someone doesn't know about that fact they won't be confused